lib/rails_admin_import/action.rb

```diff
@@ -0,0 +1,67 @@
+require "rails_admin/config/actions"
+
+module RailsAdmin
+  module Config
+    module Actions
+      class Import < Base
+        RailsAdmin::Config::Actions.register(self)
+
+        register_instance_option :collection do
+          true
+        end
+
+        register_instance_option :http_methods do
+          [:get, :post]
+        end
+
+        # FIXME: Remove. this kind of option doesn't appear in other actions
+        register_instance_option :label do
+          [:name, :id]
+        end
+
+        register_instance_option :mapping_key do
+          :name
+        end
+
+        register_instance_option :controller do
+          proc do
+            @import_model = RailsAdminImport::ImportModel.new(@abstract_model)
+
+            if request.post?
+              record_importer = RailsAdminImport::Formats.for(params[:import_format] || 'csv',
```

Comment:
Line is too long. [94/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.